src/doc/trpl/dining-philosophers.md

```diff
@@ -139,7 +139,7 @@ short strings anyway.
 One last thing you’ll notice: we just define a `Philosopher`, and seemingly
 don’t do anything with it. Rust is an ‘expression based’ language, which means
 that almost everything in Rust is an expression which returns a value. This is
-true of functions as well, the last expression is automatically returned. Since
+true of functions as well... the last expression is automatically returned. Since
```

Comment:
I'm not sure either of them are the best fit here, and I wonder if this is being interpreted slightly differently by different readers: one group as "This is also true of functions, the last expression is automatically returned" and the other interpreting the "well" as "This is also true of functions because... well... the last expression is automatically returned" (hopefully that makes sense...)

I think the first reads more natural, and should have a dash instead of a comma:
"This is true of functions as well - the last expression is automatically returned."